build-rpi-raspbian/first-boot.sh

```diff
@@ -114,5 +119,9 @@ echo "OK"
 echo "$(timestamp) [openHABian] Visit the openHAB dashboard now: http://$hostname:8080"
 echo "$(timestamp) [openHABian] To gain access to a console, simply reconnect."
 echo "$(timestamp) [openHABian] First time setup successfully finished."
+sleep 10
+sh /boot/webif.sh inst_done
+sleep 10
+sh /boot/webif.sh cleanup
```

Comment:
What's the idea here. Won't the user be confused if the page is suddenly gone?
Is there any good reason to not just show the whole log until the user decides to eventually restart the system for the first time? The only thing i can think of would be the subsequent installation of a webserver which wants to use the port 80.

Reply:
the web server will be gone, that is correct, but the page is still shown in the open browser, since the refresh from iframe isn't there anymore at this stage.
the sleeps are there to make sure that the open browser will fetch the site updates, i will increase the sleep to 12 seconds, to give the browser a little more time to do so

my thought behind this, was simply to not have running any services that are not needed anymore.
but yes, in case the user is trying to access after the installation is already done he will get an 404.
Not sure if we should keep the web server running for this case, i make it your decision :)